docker-compose.yaml

```diff
@@ -2,10 +2,9 @@ version: '3'
 services:
   dscp-node:
     image: digicatapult/dscp-node:latest
-    command:
+    command: |
```

Comment:
was it working without a pipe?

Reply:
it's a formatting thing of yaml. Without the pipe it wants to bring the first line up which makes the indentation funny. This just marks it as a multilined string